roles/netbootxyz/templates/menu/netinfo.ipxe.j2

```diff
@@ -17,5 +17,14 @@ item --gap Next-server:
 item nextserver ${space} ${next-server}
 item --gap Filename:
 item filename ${space} ${netX/filename}
+
+# Proxy DHCP info
+item --gap DHCP proxy present
+iseq ${platform} pcbios && item proxy ${space} Not supported on BIOS || isset ${proxydhcp/next-server} && item proxy ${space} Yes || item proxy ${space} No
+iseq ${platform} pcbios || item --gap DHCP proxy next-server
+iseq ${platform} pcbios || item proxy-nextserver ${space} ${proxydhcp/next-server}
+iseq ${platform} pcbios || item --gap Using DHCP proxy next-server
+iseq ${platform} pcbios || iseq ${proxydhcp/next-server} ${tftp-server} && echo Yes || echo No (Using default next-server ${tftp-server})
```

Comment:
pass or fail condition, the third || condition will fail.

Reply:
As above, the PCBIOS check will be removed